fenwick/src/main/java/org/opencadc/fenwick/ArtifactSelector.java

```diff
@@ -67,15 +67,35 @@
 
 package org.opencadc.fenwick;
 
-import org.apache.log4j.Logger;
+import java.io.File;
+import java.nio.file.Path;
+import java.util.Iterator;
+
 
 /**
- * 
+ * A Selector to provide the InventoryHarvester with a means to gather include (additive) clauses to select appropriate
+ * Artifacts to be included in the metadata sync merge.
+ *
  * @author pdowler
  */
 public abstract class ArtifactSelector {
-    private static final Logger log = Logger.getLogger(ArtifactSelector.class);
+    final Path selectorConfigDir;
+
+    protected ArtifactSelector() {
+        // Read in the configuration file, and create a usable Map from it.
+        final File homeConfigDirectory = new File(String.format("%s/config/include", System.getProperty("user.home")));
 
-    protected ArtifactSelector() { 
+        // Allow some flexibility to override.  This is useful for local testing as one would normally require root
```

Comment:
Applications only have to support $HOME/config/include - the container setup is responsible for making sure /config is found in $HOME.

Reply:
This was moved to IncludeArtifacts but it looks like this still defaults to /config/include -- we don't want that in code because the existence of /config is strictly a container operator thing. The current standard for our s/w is to look in $HOME/config only.

Reply:
Understood.  If it doesn't exist would we throw an Exception?